### DISCUSSION TITLE
Special characters in Deflayer

### DISCUSSION THREAD

**Discussion by EnriqueEkizz** [kmonad/kmonad#930] [Q&A]
Hello, when I set special characters in a "deflayer" I get either key jumps or incidents. I attach my example:

(deflayer p
  grv   !    "    #    $    %    &    /    (    )    =    '    ¿    bspc
...

For the character   "   i get a jump to the next key, "#" is printed.
For the character   (   i get the incident "unexpected '('.
For the character   (  i've tried with   "\\("   and  "\\\("   in both cases i got a issue.

I need to have the symbols in front, and when pressing shift, get the numbers only for the top number row.

I appreciate the contribution of the team.

**Comment by jokesper** [comment]
> For the character ( i've tried with "\(" and "\(" in both cases i got a issue.

You need to escape both `(` and `)`.
It should be:
```clojure
(deflayer p
grv ! " # $ % & / \( \) = ' ¿ bspc
...
```

> For the character " i get a jump to the next key, "#" is printed.

You probably use a non US layout in your OS. KMonad remaps keycodes, so we do our mapping prior to the operating system.
The names are based on the US keys.
